src/main/kotlin/no/nav/tilleggsstonader/sak/googlemaps/RouteResponse.kt

```diff
@@ -107,3 +112,27 @@ enum class RouteLabel {
     FUEL_EFFICIENT,
     SHORTER_DISTANCE,
 }
+
+enum class Maneuver {
+    MANEUVER_UNSPECIFIED,
+    TURN_SLIGHT_LEFT,
+    TURN_SHARP_LEFT,
+    UTURN_LEFT,
+    TURN_LEFT,
+    TURN_SLIGHT_RIGHT,
+    TURN_SHARP_RIGHT,
+    UTURN_RIGHT,
+    TURN_RIGHT,
+    STRAIGHT,
+    RAMP_LEFT,
+    RAMP_RIGHT,
+    MERGE,
+    FORK_LEFT,
+    FORK_RIGHT,
+    FERRY,
+    FERRY_TRAIN,
```

Comment:
Må vi ta hensyn til FERRY_TRAIN? Evt. noen av de andre her også, men de virker mer kjørerelevante

Reply:
FERRY_TRAIN er "the train leading onto the ferry". Vi vil fjerne det strekket hvor man har bilen på ferja da tiltaksenheten ikke skal betale kjørgodtgjørelse for dette. Så tar ikke med FERRY_TRAIN ☺️